sdk/identity/azure-identity/README.md

```diff
@@ -61,15 +57,18 @@ service client to authenticate requests. Service clients across the Azure SDK
 accept a credential instance when they are constructed, and use that credential
 to authenticate requests.
 
-The Azure Identity library focuses on OAuth authentication with Azure Active
-Directory (AAD). It offers a variety of credential classes capable of acquiring
-an AAD access token. See the [Credential classes](#credential-classes "Credential classes") section below for a list of
+The Azure Identity library focuses on OAuth authentication with Azure AD. It offers a variety of credential classes capable of acquiring
+an Azure AD access token. See the [Credential classes](#credential-classes "Credential classes") section below for a list of
 this library's credential classes.
 
 ### DefaultAzureCredential
 
 `DefaultAzureCredential` is appropriate for most applications which will run in the Azure Cloud because it combines common production credentials with development credentials. `DefaultAzureCredential` attempts to authenticate via the following mechanisms in this order, stopping when one succeeds:
```

Comment:
```suggestion
`DefaultAzureCredential` is appropriate for most applications which will run in Azure because it combines common production credentials with development credentials. `DefaultAzureCredential` attempts to authenticate via the following mechanisms, in this order, stopping when one succeeds:
```